src/features/game/events/landExpansion/upgradeFarm.ts

```diff
@@ -432,21 +425,62 @@ export function expireItems({
   return game;
 }
 
-function desertUpgrade(_: GameState) {
-  throw new Error("Not implemented");
+function desertUpgrade(state: GameState) {
+  const game = cloneDeep(state) as GameState;
+  // Clear the house
+  delete game.inventory["Town Center"];
+  delete game.inventory["House"];
```

Comment:
/beg Please don't empty our houses on upgrade. Placing takes hours b/c design mode is so slow.

Reply:
@brian-man The placed items in house are stored under `home.collectibles`. This code is simply swapping out type of house your inventory. It doesn't touch the contents.

The `upgradeFarm` action doesn't touch `home.collectibles` these items.

Reply:
Whew.